Ms2Database/Program.cs

```diff
@@ -1,18 +1,18 @@
-﻿using System;
-using Ms2Database.Controllers;
-using Ms2Database.DbClasses;
+﻿using Ms2Database.Controllers;
 
 namespace Ms2Database
 {
     class Program
     {
-        static void Main(string[] args)
+        static void Main(/*string[] args*/)
```

Comment:
Is there a reason for commenting out args or the manager objects?

Reply:
Yes the args were unused, but unsure if they would be used in the future so left them commented out. For the managers, they are also unused as the methods were changed to static.

Reply:
I guess this works. I'm not a huge fan of leaving code around commented out, but I guess it makes some sense.  Kind of sounds like we won't be using those objects anymore though.

Reply:
I am not familiar with the DB at all so I wasn't sure if they would eventually access member data and no longer be able to be static methods